x-pack/plugins/index_lifecycle_management/public/sections/edit_policy/components/warm_phase/warm_phase.js

```diff
@@ -202,6 +202,12 @@ class WarmPhaseUi extends PureComponent {
                             await setPhaseData(PHASE_REPLICA_COUNT, e.target.value);
                           }}
                           min={0}
+                          placeholder={
+                            intl.formatMessage({
+                              id: 'xpack.indexLifecycleMgmt.warmPhase.keepSamePlaceholder',
+                              defaultMessage: 'Keep same'
```

Comment:
A screenreader will read "Keep same" whether the user has entered a value or not. I wonder if this might be confusing and if it might be a bit of a leap for people to infer that this means "By default, the number of replicas is the same as in the index being rolled over." We could address this by using `helpText` instead, and by describing the default behavior with a complete sentence.